packages/indexer/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:18-alpine AS build
+WORKDIR /app
+ENV REDIS_URL ""
+ENV IPFS_URL ""
+ENV MAINNET_PROVIDER_URL ""
+COPY --link . .
+RUN npm install
```

Just to not have a chance to generate different images because of this step:
```suggestion
RUN npm ci
```